### PR TITLE
use new ECS modules and move to fargate

### PR DESCRIPTION
**Pull request by mgeerdsen**
### What is this PR trying to achieve?

In order to make things/maintenance less complex in the long term and to be able to easily use Secretsmanager etc., we are switching to new ECS modules and directly to using fargate.

### Who is this change for?

Those who will have to take care of the infra and want to make things more secure soon.

### CODE REVIEW

**Comment by mgeerdsen** [comment]
This is working on staging so far, but would benefit from a bit more work and refactoring to possibly reduce code duplication e.g.

**Comment by mgeerdsen** [comment]
Just as a note... a small EC2 instance has been added (named access host), which can be use to ssh in and check/modify things on EFS when needed. EFS has been quickly mounted manually, but should be added to the code using cloud-init userdata e.g.

File: infrastructure/staging/ec2_host.tf [hunk 1]
```diff
@@ -0,0 +1,67 @@
+resource "aws_instance" "access_host" {
+  ami           = data.aws_ami.amazon-linux-2.id
+  instance_type = "t2.micro"
+  key_name      = "wellcomedigitalworkflow"
```
Comment (kenoir) [inline infrastructure/staging/ec2_host.tf:4]: You might want to to turn this into a `var`.

Comment (mgeerdsen) [reply infrastructure/staging/ec2_host.tf]: instance_type and key_name are variables now, defaulting to those values.

**Pull request merged**